### PR TITLE
feat: create ProjectionTogglePane for ProposalDetailsView & makes panes more concise

### LINKED ISSUE #2065: ️🛠 - Create a projected votes pane for the proposal details view
### Task description

Moving the projected votes toggle to its own pane with title and description

### Requirements

Move projected votes toggle to its own pane on the proposal details view

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary

closes #2065 

### Platforms

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [x] Windows

## Checklist

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [x] I have verified that new and existing unit tests pass locally with my changes
-   [x] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte [hunk 1]
```diff
@@ -1,21 +1,58 @@
...
+    }
 </script>
 
-<Pane classes="p-6 flex flex-col h-fit">
-    <header-container class="flex justify-between items-center mb-4">
-        <ProposalStatusPill {proposal} />
+<div class="flex flex-col h-fit">
+    <header-container class="flex justify-between items-start mb-4">
+        <Text type="h5">{proposal?.title}</Text>
         <ProposalDetailsMenu {proposal} modalPosition={{ right: '24px', top: '54px' }} />
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte:26]:
```suggestion
        <ProposalDetailsMenu {proposal} />
```

File: packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte [hunk 2]
```diff
@@ -1,21 +1,58 @@
...
     </header-container>
-    <div class="flex flex-1 flex-col space-y-4 justify-between scrollable-y">
-        <Text type="h4">{proposal?.title}</Text>
+    <div class="flex flex-1 flex-col space-y-4 justify-between">
+        <div class="flex">
+            <ProposalStatusPill {proposal} />
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte:30]: do we want the `remainingTime` pill here as well?

File: packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte [hunk 3]
```diff
@@ -1,21 +1,58 @@
...
+        </div>
         {#if proposal?.additionalInfo}
-            <MarkdownBlock text={proposal?.additionalInfo} />
+            {@const truncate = proposal?.additionalInfo.length >= TEXT_LENGTH_TO_TRUNCATE}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte:33]: lets make sure `proposal` is not undefined. Maybe by wrapping the `ProposalDetailsView` with a `{#if $selectedProposal}`

**Comment by nicole-obrien** [comment]
Can we box the content like we do in collectibles, I will get the design updated
Screenshot 2024-03-05 at 12 38 59

**Pull request closed without merging**